index.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Options as Opts, Dropin } from 'braintree-web-drop-in';
+
+export type Options = Omit<Opts, 'container'>;
+export type DropinInstance = Dropin;
```

Comment:
What's the use for this?

Reply:
```
<DropIn
  options={braintreeOptions}
  onInstance={instance => {
    this.setBraintreeInstance(instance); <---
```

I exported it in order to provide a type for the instance, so that I could type the instance argument of `setBraintreeInstance`.

I have installed `@types/braintree-web-drop-in` as a dev dependency, but I assume that it also needs to be added as a peer dependency for this to work as intended? If yes, I can add it to the list of peer deps.